site-src/geps/gep-684.md

```diff
@@ -0,0 +1,37 @@
+# GEP-684: GEP template
+
+* Issue URL: https://github.com/kubernetes-sigs/gateway-api/pull/684
```

Comment:
This is in conlfict with the rule laid out above:
```
File an issue that will be used to track the GEP progress. Use the existing
issue templates. Use this number for naming the GEP (i.e. issue #123 would be
named `gep-123.md`).
```

We should have the first GEP as referential as possible rather than having special case here.
Can you open an issue and update the link here?